OptimizelySDK/OptimizelyFactory.cs

```diff
@@ -128,6 +139,8 @@ public static Optimizely NewDefaultInstance(string sdkKey, string fallback, stri
                 .WithSdkKey(sdkKey)
                 .WithDatafile(fallback)
                 .WithLogger(logger)
+                .WithPollingInterval(PollingInterval)
+                .WithBlockingTimeoutPeriod(BlockingTimeOutPeriod)
```

Comment:
I'm confused - why don't have the config reading unlike the above: 
                **.WithPollingInterval(PollingInterval == TimeSpan.Zero ? TimeSpan.FromMilliseconds(httpProjectConfigElement.PollingInterval) : PollingInterval)**
This can have different configuration paths depending on factory method parameters. It can be confusing unless we document very clearly about them.